### ISSUE TITLE
New ggplot version

### ISSUE THREAD

**Issue by bradystoll** [NREL/MAGMA#56]
@claytonpbarrows @aliehlen1 @oconnellm @icherny44 There is a new ggplot version (2.2.0) that changed a lot of their default behavior for plotting. In particular, the ordering of stacked bars is now the reverse be default.

Which version of ggplot are people using, and should we move to the new version or wait a bit?

**Comment by claytonpbarrows** [comment]
PH project is using update_master branch. 
In general, I think you should go ahead and update to the latest version of packages, we can checkout older versions of the MAGMA repo as needed.

**Comment by aliehlen1** [comment]
Agreed. (In case it's useful, though, we did update to the latest ggplot version, too).

**Comment by bradystoll** [comment]
Ok, sounds good. I'll update that sometime soon then.
